product_brand/README.rst

```diff
@@ -39,27 +58,31 @@ Known issues / Roadmap
   the Customers/Suppliers Form View.
 * Fix smart button alignment in brand form view
 
-
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/product-attribute/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_brand%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/product-attribute/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_brand%0Aversion:%2011.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
Keep text only upto feedback.

Reply:
@JayVora-SerpentCS this text is now auto-generated by the script, so it's correct and self-mantained.

Reply:
I see, I was referring to usual OCA readme format though.

Reply:
This is the new OCA README format :wink:

You can see the new README in the template that states the automatic generation: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst